app/views/users/login.html.erb

```diff
@@ -1,46 +1,8 @@
-<% content_for :title, _('Login') %>
-<span id="badge"></span>
-<div id="login" class="container">
-  <div class="row">
-    <div class="col-sm-12">
-      <div id="brand">
-        <%= image_tag("login_logo.png") %>
-      </div><!--/#brand-->
-    </div>
-    <% if flash_inline.any? %>
-      <div class="col-sm-12">
-        <% flash_inline.each do |type, message| %>
-          <div class="alert <%= alert_class(type) %>">
-            <% header = _('Success!') if type == :success %>
-            <%= alert_header(header || "", alert_class(type)) %>
-            <%= message %>
-          </div>
-        <% end %>
-      </div>
-    <% end  %>
-    <div class="col-sm-7 col-md-6 col-lg-5 login">
-      <%= form_tag login_users_path, :id => "login-form", :class=>"form-horizontal" do %>
-          <div class="form-group">
-            <%= label_tag "login[login]", _("Username"), :class=>"col-sm-2 control-label" %>
-            <div class="col-sm-10">
-              <%= text_field :login, :login, {:class=>"form-control", :focus_on_load => true} %>
-            </div>
-          </div>
-          <div class="form-group">
-            <%= label_tag "login[password]", _("Password"), :class=>"col-sm-2 control-label" %>
-            <div class="col-sm-10">
-              <%= password_field_tag 'login[password]', nil, :class => 'form-control' %>
-              <span class="glyphicon glyphicon-warning-sign input-addon"
-                title="<%= _('Caps lock ON') %>" style="display:none"></span>
-            </div>
-          </div>
-          <div class="form-group">
-            <div class="col-xs-offset-8 col-xs-4 submit">
-              <%= submit_tag(_("Log In").html_safe, :class => "btn btn-primary", :data => { :disable_with => _("Log In") }) %>
-            </div>
-          </div>
-      <% end %>
-    </div>
-    <%= render 'welcome_box' %>
-  </div>
-</div>
+<div class="login-page">
+<%= mount_react_component('ComponentWrapper', '.login-page', {
+    component: 'LoginPage',
+    componentProps: {
+        token: form_authenticity_token,
+        version: SETTINGS[:version],
```

Comment:
maybe pass the current locale? (so you could show it in the correct language and have it selected properly?)

Reply:
on a follow up PR I will add the language selector support